weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>I assume you mean <code>Explorer</code> on the cloud console - but I think it is worth clarifying here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>